packages/api/lib/writeRecords/write-granules.js

```diff
@@ -869,8 +870,13 @@ const writeGranulesFromMessage = async ({
       const updatedAt = now;
       const timestamp = now;
 
+      let published = granule.published;
+      if (published === undefined) {
+        published = false;
+      }
```

Comment:
Do we need to handle `null` differently here like we do with PATCHing files? Or handle it at all I suppose. If it's null or undefined, published = false? I think you can have a `null` published in Dynamo?

Reply:
Make sure unit covers null

Reply:
@npauzenga  let me know what you think: c2c20b6f0d